cli/azd/appdetect/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/azure/azure-dev/cli/azd/internal/appdetect"
+)
+
+var directory string
+var projectDirectory string
+
+func init() {
+	flag.StringVar(&directory, "dir", "", "Directory containing projects to detect")
+	flag.StringVar(&projectDirectory, "proj", "", "Specific project directory to detect")
+}
+
+func main() {
```

Comment:
Was thinking this could be a useful tool to just test it against any real-world projects, but if it's too much we can also just not have this for now.

Reply:
I like these little tools!